docs/html-support.md

```diff
@@ -0,0 +1,121 @@
+# Enable HTML Templates in EmailOTP
+
+Use this configurations if you need to use HTML Templates in the Emails.
+**This feature is supported from Identity Server v5.6 onwards**.
+
+1. Configure Email settings in \<IS_HOME>/repository/conf/output-event-adapters.xml file. 
+```
+<adapterConfig type="email">
+    <!-- Comment mail.smtp.user and mail.smtp.password properties to support connecting SMTP servers which use trust
+        based authentication rather username/password authentication -->
+    <property key="mail.smtp.from">abcd@gmail.com</property>
+    <property key="mail.smtp.user">abcd</property>
+    <property key="mail.smtp.password">xxxx</property>
+    <property key="mail.smtp.host">smtp.gmail.com</property>
+    <property key="mail.smtp.port">587</property>
+    <property key="mail.smtp.starttls.enable">true</property>
+    <property key="mail.smtp.auth">true</property>
+    <!-- Thread Pool Related Properties -->
+    <property key="minThread">8</property>
+    <property key="maxThread">100</property>
+    <property key="keepAliveTimeInMillis">20000</property>
+    <property key="jobQueueSize">10000</property>
+</adapterConfig>
+```
+2. Open \<IS_HOME>/repository/conf/identity/application-authentication.xml and add the following parameter into 
```

Comment:
Shall we say:
Open the <IS_HOME>/repository/conf/identity/application-authentication.xml and add the following parameter under <AuthenticatorConfig name="EmailOTP" enabled="true">:

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/30/commits/f578717bad3139812d57d434f1d4275141d260be